vendor/modules.txt

```diff
@@ -397,6 +400,9 @@ github.com/prometheus/common/model
 github.com/prometheus/procfs
 github.com/prometheus/procfs/internal/fs
 github.com/prometheus/procfs/internal/util
+# github.com/reflog/dateconstraints v0.2.1
+## explicit
+github.com/reflog/dateconstraints
```

Comment:
You need to add the changes to include in the NOTICE.txt file the licenses that applies to this new packages included. This one and the `Masterminds/semver/v3` one.